src/components/creditNote/CreditNotesTable.tsx

```diff
@@ -0,0 +1,418 @@
+                  }),
+                )
+              }}
+              columns={[
+                {
+                  key: 'totalAmountCents',
+                  title: translate('text_1727078012568v9460bmnh8a'),
+                  content: (creditNote) => <CreditNoteBadge creditNote={creditNote} />,
+                },
+                {
+                  key: 'number',
+                  title: translate('text_64188b3d9735d5007d71227f'),
+                  minWidth: 160,
+                  content: ({ number }) => (
+                    <Typography variant="body" noWrap>
+                      {number}
+                    </Typography>
+                  ),
+                },
+                {
+                  key: 'totalAmountCents',
+                  title: translate('text_62544c1db13ca10187214d85'),
+                  content: ({ totalAmountCents, currency }) => (
+                    <Typography
+                      className="font-medium"
+                      variant="body"
+                      color={showCustomerName ? 'grey700' : 'success600'}
+                      align="right"
+                      noWrap
+                    >
+                      {intlFormatNumber(deserializeAmount(totalAmountCents || 0, currency), {
+                        currencyDisplay: 'symbol',
+                        currency,
+                      })}
+                    </Typography>
+                  ),
+                  maxSpace: !showCustomerName,
+                  textAlign: 'right',
+                },
+                ...(showCustomerName
+                  ? [
+                      {
+                        key: 'invoice.customer.displayName',
+                        title: translate('text_63ac86d797f728a87b2f9fb3'),
+                        content: (creditNote: CreditNoteTableItemFragment) => (
+                          <CustomerColumn variant="body" color="grey600" noWrap>
+                            {creditNote.invoice?.customer.displayName}
+                          </CustomerColumn>
+                        ),
+                        maxSpace: true,
+                      } as TableColumn<CreditNoteTableItemFragment>,
+                    ]
+                  : []),
+                {
+                  key: 'createdAt',
+                  title: translate('text_62544c1db13ca10187214d7f'),
+                  content: ({ createdAt }) => (
+                    <Typography variant="body" color="grey600" noWrap>
+                      {customerTimezone
+                        ? formatDateToTZ(createdAt, customerTimezone)
+                        : formatTimeOrgaTZ(createdAt)}
+                    </Typography>
+                  ),
+                },
+              ]}
+            />
+          </InfiniteScroll>
+        )}
+      </div>
+
+      <VoidCreditNoteDialog ref={voidCreditNoteDialogRef} />
+    </ScrollContainer>
+  )
+}
+
+export default CreditNotesTable
+
+const ScrollContainer = styled.div`
```

Comment:
Do you think you can merge to tailwind?

Reply:
That's a good idea, but I say we keep it as a later improvement. Quite a lot of styled components in the file, and migrating them could break things and would require another QA session.

Reply:
Ok no problem!